mono/utils/mono-mmap-windows.c

```diff
@@ -142,25 +142,18 @@ mono_file_map_error (size_t length, int flags, int fd, guint64 offset, void **re
 	void *ptr = NULL;
 	HANDLE mapping = NULL;
 	int const prot = mono_mmap_win_prot_from_flags (flags);
-	/* translate the flags */
```

Comment:
Revert that removal.

Reply:
I restored the first line, which remained accurate. I kept removed the dead code. PR feedback is ambiguous to me as to if the entire removal or just the first part was referred to.